content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md

```diff
@@ -236,7 +243,7 @@ The following command specifies that the `CONFIG`, `RBAC` and `CRYPTO` schema sh
    {{< /tabs >}}
 
 {{< note >}}
-If the schemas are not specified, then the tables will be created in the default schema and the next steps in this procedure will need updating to reflect this.
+In the case of schemas, `CONFIG` applies to database and crypto and `RBAC` applies to create-user-config. If the schemas are not specified, then the tables will be created in the default schema and the next steps in this procedure will need updating to reflect this.
```

Comment:
Perhaps consider:
`If you are targeting schemas, database and crypto generated SQL should be applied to the CONFIG schema, and create-user-config generated SQL should be applied to the RBAC schema.`

i.e. I think the target and subject of the "applies" is the wrong way round at the moment